tests.py

```diff
@@ -59,28 +57,46 @@
 random.seed(42)
 
 test_operator_key = 'sklearn.feature_selection.SelectKBest'
-TPOTSelectKBest,TPOTSelectKBest_args = TPOTOperatorClassFactory(test_operator_key,
-                                            classifier_config_dict[test_operator_key])
+TPOTSelectKBest, TPOTSelectKBest_args = TPOTOperatorClassFactory(
+    test_operator_key,
+    classifier_config_dict[test_operator_key]
+)
+
+
+# http://stackoverflow.com/questions/5595425/
+def is_close(a, b, rel_tol=1e-09, abs_tol=0.0):
```

Comment:
We should use numpy.allclose here: https://docs.scipy.org/doc/numpy/reference/generated/numpy.allclose.html